.github/workflows/ci.yml

```diff
@@ -20,24 +24,52 @@ jobs:
       - name: Install Go
         uses: actions/setup-go@v3
         with:
-          go-version: 1.17.x
-      - name: Retrieve get module name
-        run: |
-          echo "::set-output name=Module::$(go list -m)"
-        id: module-name
-      - name: Verify builds with xk6
+          go-version: 1.20.x
+      - name: Check build
         run: |
-          go install go.k6.io/xk6/cmd/xk6@latest
-          xk6 build --with ${{ steps.module-name.outputs.Module }}=.
+          go version
+          pwd && ls -l
+          
+          go install go.k6.io/xk6/cmd/xk6@master
+          MODULE_NAME=`go list -m`
+          
+          GOPRIVATE="go.k6.io/k6" xk6 build \
+            --output ./k6ext \
+            --with $MODULE_NAME="."
```

Comment:
I think this was a temporary workaround for avoiding the Go proxy cache, since it had some stale version of k6 because of a bad release on our side, but it shouldn't be needed anymore.